web/ui/react-app/src/pages/graph/CMTheme.tsx

```diff
@@ -239,3 +277,19 @@ export const promqlHighlighter = HighlightStyle.define([
   { tag: tags.invalid, color: 'red' },
   { tag: tags.comment, color: '#888', fontStyle: 'italic' },
 ]);
+
+export const darkPromqlHighlighter = HighlightStyle.define([
+  { tag: tags.name, color: '#000' },
```

Comment:
Whoops, we just found out that this caused https://github.com/prometheus/prometheus/issues/11568. We are accidentally setting the text color to black on black here. Will send a fix.